src/github.com/matrix-org/go-neb/services/jira/webhook/webhook.go

```diff
@@ -153,22 +155,18 @@ func createWebhook(jrealm *jira.Realm, webhookEndpointURL, userID string) error
 	return err
 }
 
-func getWebhook(cli *gojira.Client, webhookEndpointURL string) (*jiraWebhook, *util.HTTPError) {
+func getWebhook(cli *gojira.Client, webhookEndpointURL string) (*jiraWebhook, bool, error) {
```

Comment:
I think you should add some comments to explain that the bool returned means "forbidden".